bot/exts/gurkan/make_gurkan.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+import discord
+from bot.bot import Bot
+from bot.constants import Roles
+from discord.ext import commands
+from discord.utils import get
+
+
+class MakeGurkan(commands.Cog):
+    """Make Gurkan Cog adds or removes the gurkan role depending on if the user has gurkan in their name."""
```

Comment:
```suggestion
    """Makes sure that only members with gurkan names have the gurkan role."""
```
Let's make it a bit more concise.